profile/config.go

```diff
@@ -295,7 +295,7 @@ Examples:
 		Order:           cfgOptionBlockP2POrder,
 		OptType:         config.OptTypeInt,
 		ExternalOptType: "security level",
-		DefaultValue:    status.SecurityLevelsAll,
+		DefaultValue:    status.SecurityLevelExtreme,
 		ValidationRegex: "^(4|6|7)$",
```

Comment:
should we allow the use to completely turn P2P blocking off?

Reply:
In general, only a few select settings should be able to be turned completely off. The Extreme Security Level is panic mode that activates everything at the Portmaster's disposal and also does not change between systems to a certain degree, in order to make it more reliable in the way it works.